libvirt/README.md

```diff
@@ -26,13 +34,27 @@ If you want to set default password for debugging then you can use guestfish to
 
 # Running cloud-api-adaptor
 
+Export the required environment variables.
+
+```
+export LIBVIRT_URI=REPLACE_ME
+export LIBVIRT_POOL=REPLACE_ME
+export LIBVIRT_NET=REPLACE_ME
 ```
+Note that the `LIBVIRT_URI` should be of the form - `qemu+ssh://root@<LIBVIRT_HOST_ADDR>/system`.
+
+Run the binary.
+
+```
+mkdir -p /opt/data-dir
+
 ./cloud-api-adaptor libvirt \
-     -host-interface ens3 \
-     -network-name default  \
-     -pool-name default \
-     -uri "qemu+ssh://root@192.168.122.1/system" \
-     -data-dir "/var/lib/libvirt" 
+    -uri ${LIBVIRT_URI}  \
+    -data-dir /opt/data-dir \
+    -pods-dir /run/peerpod/pods \
+    -network-name ${LIBVIRT_NET} \
+    -pool-name ${LIBVIRT_POOL} \
+    -socket /run/peerpod/hypervisor.sock
```

Comment:
cri-runtime-endpoint was added

Reply:
> cri-runtime-endpoint

Yes, this is an optional parameter for CAA, so that we can retrieve the images tag from a digest if an image is pulled 2nd time in guest.

Reply:
Right - https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/pkg/adaptor/proxy/proxy.go#L149

I think for now we can avoid changing the README since its optional param.